src/billforward.js

```diff
@@ -3249,9 +3343,9 @@
     };
 
     bfjs.captureBankAccount = function(bankAccountDetails, targetGateway, accountID, callback) {
-        if (targetGateway != "stripe") {
+        if (!['stripe', 'goCardless', 'gocardless'].includes(targetGateway)) {
             throw "'" + targetGateway + "' gateway does not support bank accounts";
-        } else {
+        } else if (targetGateway === 'stripe') {
```

Comment:
Would this if/else statement make more sense in a different order:
`if (stripe) ... else if (gocardless) ... else ....` ?

Reply:
Hmm, I think all I'm going to do is separate them into two independent `if` statements. The first is a validation step ("is this a valid gateway?") and the second is some kind of trick with the Stripe gateway that I don't understand.

So
```js
if (!['stripe', 'goCardless', 'gocardless'].includes(targetGateway)) {
    throw "'" + targetGateway + "' gateway does not support bank accounts";
}

if (targetGateway === 'stripe') {
    // We are masking the stripe ach gateway as a plain stripe one
    targetGateway = "stripeach";
}
```